### ISSUE TITLE
React native reanimated not supported less the next version

### ISSUE THREAD

**Issue by barak109** [dannyhw/addon_react_native_web_example#1]
Hi Danny, thanks for the great example.

I took your repo and just changed the version of react-native-reanimted to the version we have: 2.2.4 and the animated doesn't work.

I tried to check it with 2.3.0 and many other versions between 2.0.0 to the latest stable version 2.3.0.

React native reanimated said that they support from version 2.0.0 in web.

No one of the other versions works, only the 'next' one (2.8.0) that you use in this repo.

We can't update to 2.8.0 because it causes breaking changes in our big app, I will appreciate if you have a solution for us.

Thanks in advance.

**Comment by dannyhw** [comment]
@barak109 hey! I've not updated this example in a little while, let me update everything and I can get it working for you.

**Comment by barak109** [comment]
> @barak109 hey! I've not updated this example in a little while, let me update everything and I can get it working for you.

Thanks Danny, I appreciate it!

**Comment by dannyhw** [comment]
@barak109 I just updated everything in the project to the latest versions. I'm not sure about the reanimated problems you have though can you explain a bit more?

**Comment by dannyhw** [comment]
I had used that version of reanimated because there was some bug with the stable version at the time. However i think its resolved in newer versions.

**Comment by barak109** [comment]
> @barak109 I just updated everything in the project to the latest versions. I'm not sure about the reanimated problems you have though can you explain a bit more?

@dannyhw  Yes, the problem is that we can't use the version you used with react-native-reanimated ( next = 2.8.0) so I'm asking if you know how to get it to work with 2.2.4 version for reanimated the one we have in the project?

Maybe there is really a bug as you said in this version.

**Comment by dannyhw** [comment]
@barak109 well what I mean is what error do you have? what is making it not work?

**Comment by barak109** [comment]
> @barak109 well what I mean is what error do you have? what is making it not work?

@dannyhw , I’m getting this error
https://github.com/software-mansion/react-native-reanimated/issues/2546

**Comment by dannyhw** [comment]
so I think the version 2.2.4 had some issues with web but 2.3.3 works fine, does that version not work for you?
